### PR TITLE
refactor(frontend):  toast by automatically logged out

### LINKED ISSUE #2675: 🔧 [Refactor] Toast Message when forced to logout clientside
## 🔧 Refactor ticket

Toast Message when forced to logout clientside.

When the client forces a logout the user should get a toast informing him about the enforced logout.
The message should be something like `Your session has expired - you were logged out`

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
When automatically logging out a message is displayed that automatically logged out. this message remains and is not closed after 5 seconds. 
i have created a variable in the store **automaticallyLoggedOut** . this is set to true when logging out automatically. and when login.vue is called in **created** augefragt and the toast message is set. after that the variable is set to false again. also with a login the variable in the store is automatically set to false again. 

### Issues
- fixes #2675 

https://user-images.githubusercontent.com/1324583/217480763-ceef2b77-891c-4a28-8fef-42760c16d67b.mp4

### CODE REVIEW

**Review by ulfgebhardt — APPROVED** [review]

**Review by Mogge — CHANGES_REQUESTED** [review]

File: frontend/src/components/SessionLogoutTimeout.vue [hunk 1]
```diff
@@ -67,6 +67,7 @@ export default {
       }
       if (this.tokenExpiresInSeconds === 0) {
         this.$timer.stop('tokenExpires')
+        this.toastInfoNoHide('Du wurdest automatisch abgemeldet')
```
Comment (Mogge) [inline frontend/src/components/SessionLogoutTimeout.vue:70]: Localize!

File: frontend/src/components/SessionLogoutTimeout.vue [hunk 2]
```diff
@@ -84,6 +85,7 @@ export default {
         })
         .catch(() => {
           this.$timer.stop('tokenExpires')
+          this.toastInfoNoHide('Du wurdest automatisch abgemeldet')
```
Comment (Mogge) [inline frontend/src/components/SessionLogoutTimeout.vue:88]: Localize!

**Review by Mogge — CHANGES_REQUESTED** [review]

File: frontend/src/mixins/toaster.js [hunk 3]
```diff
@@ -4,24 +4,33 @@ export const toasters = {
...
+      this.toast(message, {
+        title: this.$t('navigation.info'),
+        variant: 'warning',
+        noAutoHide: true,
       })
     },
     toast(message, options) {
       if (message.replace) message = message.replace(/^GraphQL error: /, '')
       this.$root.$bvToast.toast(message, {
-        autoHideDelay: 5000,
```
[the remark this answers is not in the document]
Comment (ogerly) [reply frontend/src/mixins/toaster.js:24]: done

File: frontend/src/mixins/toaster.js [hunk 4]
```diff
@@ -4,24 +4,33 @@ export const toasters = {
...
       })
     },
     toastInfo(message) {
       this.toast(message, {
         title: this.$t('navigation.info'),
         variant: 'warning',
+        autoHideDelay: 5000,
+      })
+    },
+    toastInfoNoHide(message) {
```
[the remark this answers is not in the document]
Comment (ogerly) [reply frontend/src/mixins/toaster.js:21]: done

**Review by Mogge — DISMISSED** [review]

File: frontend/src/components/SessionLogoutTimeout.vue (hunk 1 above)
Comment (ogerly) [reply frontend/src/components/SessionLogoutTimeout.vue]: done

File: frontend/src/components/SessionLogoutTimeout.vue (hunk 2 above)
Comment (ogerly) [reply frontend/src/components/SessionLogoutTimeout.vue]: done

**Review by ulfgebhardt — APPROVED** [review]

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**